docs/section10/10-9.md

```diff
@@ -0,0 +1,65 @@
+# Airside systems – Rule 10-9  
+**Schema Version:** 0.0.17    
+**Mandatory Rule:** True     
+**Rule ID:** 10-9  
+ 
+**Rule Description:** Where multiple HVAC zones or residential spaces are combined into a single thermal block, baseline HVAC System Types 5 or 6 efficiencies shall be based on the cooling equipment capacity of a single floor.  
+
+**Rule Assertion:** B-RMR = expected value                                             
+**Appendix G Section:** Section G3.1.2.1  
+**Appendix G Section Reference:** None  
+**Data Lookup:** None   
+**Evaluation Context:** Evaluate each HeatingVentilationAirAconditioningSystem   
+
+**Applicability Checks:**  
+
+1. Applicable to baseline HVAC system types 5 or 6 that include more than one floor.  
+
+**Function Call:**  
+
+1. get_baseline_system_types()
+2. get_hvac_systems_5_6_serving_multiple_floors_b()
+
+**Manual Check:** None.  
+ 
+## Rule Logic:  
+**Applicability Check 1:**  
+- Get baseline hvac system type dictionary: `baseline_hvac_sys_type_ids_dict_b = get_baseline_system_types(B_RMR)`
+- Get dictionary with hvac system ids serving more than one floor as keys and the number of floors served as values: `hvac_sys_serve_more_than_one_flr_dict_b = get_hvac_systems_5_6_serving_multiple_floors_b(B_RMR)`
+- Get list of hvac systems of type 5 and 6 that serve more than one floor: `hvac_sys_serve_more_than_one_flr_list_b = hvac_sys_serve_more_than_one_flr_dict_b.keys()`
+- For each hvac_b in the B_RMR: `for hvac_b in the B_RMR...HeatingVentilationAirConditioningSystem:`     
+    - Check if hvac_b is in hvac_sys_serve_more_than_one_flr_list_b (outcome is Not Applicable for all HVAC systems not in this list): `if hvac_b in hvac_sys_serve_more_than_one_flr_list_b:`  
+        - Get the number of floors that the hvac system serves: `num_floors_hvac_b = hvac_sys_serve_more_than_one_flr_dict_b[hvac_b.id]`
+        - Reset the efficiency found variable to false, if efficiency is not reported in the correct units for defining the baseline then the outcome of this RDS is fail: `was_efficiency_found = FALSE`
```

Comment:
I wonder if we need this variable. If it is deleted, then we can also remove line 45 and also take it out of Case 1 assertion.